src/Microsoft.SqlTools.ServiceLayer/Agent/JobUtilities.cs

```diff
@@ -123,5 +73,46 @@ public static AgentJobStep ConvertToAgentJobStep(DataRow jobRow, SqlConnectionIn
             step.RunStatus = runStatus;
             return step;
         }
+
+        public static List<AgentJobHistoryInfo> ConvertToAgentJobHistoryInfo(List<ILogEntry> logEntries, DataRow jobRow) {
```

Comment:
`{` should be on its own line.  Please make this change for all `{` characters in the PR.